irc/handlers.go

```diff
@@ -2157,6 +2157,27 @@ func dispatchMessageToTarget(client *Client, tags map[string]string, histType hi
 			return
 		}
 		channel.SendSplitMessage(command, lowestPrefix, tags, client, message, rb)
+	} else if target[0] == '$' && len(target) > 2 && client.Oper().HasRoleCapab("massmessage") {
+		details := client.Details()
+		matcher, err := utils.CompileGlob(target[1:], false)
```

Comment:
Could you make this `target[2:]` and then choose which string to match against, instead of the `fmt.Sprintf` technique?